payments-model/src/test/java/com/stripe/android/repository/ConsumersApiServiceImplTest.kt

```diff
@@ -0,0 +1,75 @@
+package com.stripe.android.repository
+
+import com.stripe.android.ConsumerFixtures
+import com.stripe.android.core.ApiVersion
+import com.stripe.android.core.networking.ApiRequest
+import com.stripe.android.core.networking.StripeNetworkClient
+import com.stripe.android.core.networking.StripeResponse
+import com.stripe.android.core.version.StripeSdkVersion
+import kotlinx.coroutines.test.runTest
+import org.junit.Assert.assertEquals
+import org.junit.Test
+import org.mockito.Mockito.verify
+import org.mockito.kotlin.KArgumentCaptor
+import org.mockito.kotlin.any
+import org.mockito.kotlin.argumentCaptor
+import org.mockito.kotlin.mock
+import org.mockito.kotlin.whenever
+
+class ConsumersApiServiceImplTest {
+
+    private val stripeNetworkClient: StripeNetworkClient = mock()
+
+    private val consumersApiService = ConsumersApiServiceImpl(
+        stripeNetworkClient = stripeNetworkClient,
+        sdkVersion = StripeSdkVersion.VERSION,
+        apiVersion = ApiVersion(betas = emptySet()).code,
+        appInfo = null
+    )
+
+    private val apiRequestArgumentCaptor: KArgumentCaptor<ApiRequest> = argumentCaptor()
+
+    @Test
+    fun `lookupConsumerSession() sends all parameters`() = runTest {
```

Comment:
This looks like it could benefit from a project I'm working on :) https://github.com/stripe/stripe-android/pull/6115
Not blocking, but I'd be happy to tell you about it offline if you'd like!

Reply:
💯 💯 💯